socorro/submitter/submitter_app.py

```diff
@@ -158,15 +152,6 @@ def new_crashes(self):
             yield (((prefix, crash_pathnames), ), {})
 
 
-# this class was relocated to a more appropriate module and given a new name.
-# This import is offered for backwards compatibilty.  Note, that there has also
-# been an internal change to the required config, with the source
-# implementation moved into a namespace
-from socorro.external.postgresql.new_crash_source import (
-    DBCrashStorageWrapperNewCrashSource as DBSamplingCrashSource
-)
```

Comment:
Nothing uses this name from this module, so I nixed this.

Reply:
👍